api/apps/api/src/modules/specification/calculate-features/application/calculate-features-for-specification.handler.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  CommandHandler,
+  EventPublisher,
+  IInferredCommandHandler,
+} from '@nestjs/cqrs';
+import { Logger } from '@nestjs/common';
+
+import { SpecificationRepository } from './specification.repository';
+
+import { CalculateFeaturesForSpecification } from '../../commands/calculate-features-for-specification.command';
+
+@CommandHandler(CalculateFeaturesForSpecification)
+export class CalculateFeaturesForSpecificationHandler
+  implements IInferredCommandHandler<CalculateFeaturesForSpecification> {
+  private readonly logger: Logger = new Logger(
+    CalculateFeaturesForSpecificationHandler.name,
+  );
+
+  constructor(
+    private readonly eventPublisher: EventPublisher,
+    private readonly specificationRepository: SpecificationRepository,
+  ) {}
+
+  async execute({
+    featureIds,
+    specificationId,
+  }: CalculateFeaturesForSpecification): Promise<void> {
+    let specification = await this.specificationRepository.getById(
+      specificationId,
+    );
+    if (!specification) {
+      this.logger.warn(`Couldn't find specification: ${specificationId}`);
+      return;
+    }
```

Comment:
I'd say it's an error

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/417 via 
* https://github.com/Vizzuality/marxan-cloud/pull/417/commits/b0a568064ca61b4f26ea5aa3d4e3fe0cf6e39563
* https://github.com/Vizzuality/marxan-cloud/pull/417/commits/9b8b6dd9e1d20e8c91c1f5ba739f215820a4678d